docs/skills-client/common/slillsDisplayTheme.md

```diff
@@ -29,15 +37,15 @@ The theme object supports the following properties:
 | charts.axisLabelColor | color of scales/axis text on charts | 
 | tiles.backgroundColor | tile background color, such as subject tiles | 
 | tiles.watermarkIconColor | My Rank, Rank overview, My Badges watermark icon color | 
-| buttons.backgroundColor <since project="skills-service" version="1.4.0" :is-block="true"/> | background for buttons | 
-| buttons.foregroundColor <since project="skills-service" version="1.4.0" :is-block="true"/> | foreground color for buttons including text and border |
-| buttons.disabledColor <since project="skills-service" version="1.4.0" :is-block="true"/> | disabled color for buttons including text and border |
+| buttons.backgroundColor  | background for buttons | 
```

Comment:
should use language consistent with foregroundColor/disabledColor theme settings

Reply:
:+1: